src/execSync.js

```diff
@@ -1,7 +1,9 @@
 import child from "child_process";
 
-export default function execSync(cmd) {
-  return child.execSync(cmd, {
-    encoding: "utf8"
-  }).trim();
+export default function execSync(cmd, options = {}) {
+  if (!options.encoding) {
+    options.encoding = "utf8";
```

Comment:
If anything, this should be the default, with the `options` assigned over (not mutating the parameter value).

```js
const opts = Object.assign({
  encoding: "utf8",
}, options);

return child.execSync(md, opts).trim();
```